docs/development/tests.rst

```diff
@@ -89,7 +52,6 @@ or skip some (like search tests)::
 
 Current marks are:
 
-- search (tests that require Elastic Search)
```

Comment:
Why are we removing the `search` mark? Didn't we agree that we were creating `proxito` and `search` tox testenv by sharing the environment itself to speed up?

I definitely don't want to run `search` tests locally when running my app tests. I don't spin search when doing docker up because it kills my computer; so all these tests fail when picked by pytest.

Reply:
I think we all agree to be able to run tests with `-k`, marks are used only for different configuration files. I'm -1 on having another environment, as it requires more typing to run the whole suite, and requires us tracking search tests, I think we don't even track those in .com.